dips/dip-1.mdx

```diff
@@ -614,24 +614,21 @@ The data that contributes to the compliance recipient signature.
 
     amount (u64) = "5123456" (Hex "802d4e0000000000")
 
-Metadata is serialized using  LCS (including encoding of `reference_id`) and appended to the fixed length byte sequences representing `address`, `amount`, and `DOMAIN_SEPARATOR`. For example the byte sequence that is signed for the transaction data above is (bytes, hex):
+Metadata is serialized using  LCS (including encoding of `reference_id`) and appended to the fixed length byte sequences representing `address`, `amount`, and `DOMAIN_SEPARATOR`. For example the byte sequence that is signed for the transaction data above is (in hex-encoded bytes):
 
-     "02000120626239393164386533653630313165623865616561636465343830303131323253414d504c4552454641444452455353802d4e0000000000404024244449454d5f41545445535424244040" (len=158)
+     "02004257855C8410735AC58E2B5C422C479453414d504c4552454641444452455353802d4e0000000000404024244449454d5f41545445535424244040" (len=122)
 
 The above serialized byte array represents:
 
         "0200"                                      - Metadata type and version (2 bytes, constant value)
-        "0120"                                      - uleb128 encoded reference_id length (variable)
-        "6262 ... 3232"                             - Bytes of reference_id (variable)
+        "4257855C8410735AC58E2B5C422C4794"          - Bytes of reference_id (16 bytes)
```

Comment:
=> reference_id (16 bytes, hex-encoded)

Reply:
I kept the format that matches the rest of this, lemme know if you think we should add the hex encoding to everything thing here

Reply:
this value needs to be tweaked to be 8bytes but the rest of the doc represents discussions thus far